pages/nfttest/mint/_iscnId.vue

```diff
@@ -272,6 +274,79 @@ export default class NFTTestMintPage extends Vue {
     }
   }
 
+  async getOgImage() {
+    try {
+      const { url } = this.iscnData.contentMetadata
+      if (!url) { return }
+      const { data: { image } } = await this.$axios.get(`/crawler/?url=${encodeURIComponent(url)}`)
+      if (!image) { return }
+      const { data, headers } = await this.$axios.get(image)
+      const blob = new Blob([data], { type: headers['content-type'] })
+      const formData = new FormData()
+      formData.append('file', blob)
+      this.ogImageFormData = formData
```

Comment:
Storing image blob as `data`, and formdata as `computed` make this more readable